httpstats/handler.go

```diff
@@ -118,5 +140,12 @@ func (w *responseWriter) complete() {
 	}
 
 	w.metrics.observeResponse(res, "write", w.bytes, now.Sub(w.start))
+
+	// UserAgent can result in high cardinality. Allow mechanism for disabling the tag
+	if w.conf.DisableUserAgent {
+		w.metrics.http.userAgent = ""
+
```

Comment:
nit

Reply:
sorry, what's the nit you're referring to?